### PR TITLE
Do not overwrite cfme data + compliance logic fix

### PR DESCRIPTION
**Pull request by mfalesni**
* write to cfme_data instead
* do the provisioning of a testing VM (wasn't done before)

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 1bc37f06815104c789529564cacd91cc4b423572:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit cc38d47e2e7cfaa8f5f5ef3ce13e6b929815074a:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./cfme/tests/control/test_compliance.py:4:1: F401 'random' imported but unused
```
*CFME QE Bot*

File: fixtures/parallelizer/remote.py [hunk 1]
```diff
@@ -213,8 +213,8 @@ def _init_config(slave_options, slave_args):
     appliance_data = conf.slave_config.get("appliance_data", {})
     if ip_address in appliance_data:
         template_name, provider_name = appliance_data[ip_address]
-        conf.runtime["cfme_data"]["basic_info"]["appliance_template"] = template_name
-        conf.runtime["cfme_data"]["basic_info"]["appliances_provider"] = provider_name
+        conf.cfme_data["basic_info"]["appliance_template"] = template_name
+        conf.cfme_data["basic_info"]["appliances_provider"] = provider_name
```
Comment (seandst) [inline fixtures/parallelizer/remote.py]: You should never write directly to a conf. Why the change?

Comment (mfalesni) [inline fixtures/parallelizer/remote.py]: when you write to runtime, everything other is forgotten (like vddk_url and so ...). I don't know whether to copy everything to runtime and then modify or what ... :cry:

Comment (seandst) [inline fixtures/parallelizer/remote.py]: That should not be happening, and is a conf bug. This might actually be fixed in the yaycl branch, so I'll get that sorted and and also get a test covering this case (which yaycl *might* already have).

Comment (mfalesni) [inline fixtures/parallelizer/remote.py]: @seandst Ok, ping me then when it is done ...

Comment (seandst) [inline fixtures/parallelizer/remote.py]: @mfalesni I've fixed this bug, pardon the delay.
